src/concurrency/index.md

```diff
@@ -383,8 +383,8 @@ to get a safe counter with no unsafe code at all!
 
 This is great for simple types like the `u32` of our counter, but what about
 more complex types which are not Copy? An extremely common example in an
-embedded context is a peripheral struct, which generally are not Copy.
-For that we can turn to `RefCell`.
+embedded context is a peripheral struct, which generally is not Copy.
```

Comment:
Thanks, it looks like `is not Copy` is the more commonly used spelling instead of `are not`.

Reply:
I just thought it must agree with the singular subject "a peripheral struct".